src/services/study/short-circuit-analysis.js

```diff
@@ -121,8 +120,9 @@ export function fetchShortCircuitAnalysisPagedResults({
     urlSearchParams.append('page', page);
 
     if (sort) {
-        const { colKey, sortWay } = sort;
-        urlSearchParams.append('sort', `${colKey},${getSortValue(sortWay)}`);
+        sort.map((value) =>
+            urlSearchParams.append('sort', `${value.colId},${value.sort}`)
+        );
     }
```

Comment:
use "sort?." like other files instead of using "if" condition

Reply:
Yeah I just wanted to change the minimum code possible